test-plugins/build.gradle

```diff
@@ -0,0 +1,5 @@
+apply from: rootProject.file('gradle/lantern.gradle')
+
+// Basic project information
+archivesBaseName = 'lantern-test-plugins'
+version = 'cero.sero.zero'
```

Comment:
@Meronat Should we also version the test plugins? I don't think so because every plugin in this directory can have a different one.